.github/workflows/ci.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
             rust: nightly
           - name: "Tests"
             cmd: nextest
-            args: run --workspace --all-features --features cacao --retries 3
+            args: run --workspace --all-features
```

Comment:
`--all-features` implies `--features cacao`

`--retries` doesn't seem to be necessary in latest version